src/Apps/Collect/Components/ArtworkGrid/index.tsx

```diff
@@ -54,6 +54,7 @@ class Filter extends Component<Props, State> {
         max={FilterState.MAX_PRICE}
         step={50}
         defaultValue={[initialMin, initialMax]}
+        disabled={filters.isWayToBuySelected()}
```

Comment:
we should only disable if `this.state.at_auction` is `true`

Reply:
Good catch :+1: